docs/guides-using-fake-timers.mdx

```diff
@@ -10,11 +10,8 @@ flaky.
 
 To solve these problems, or if you need to rely on specific timestamps in your
 code, most testing frameworks offer the option to replace the real timers in
-your tests with fake ones. This should be used sporadically and not on a regular
-basis since using it contains some overhead.
-
-When using fake timers in your tests, all of the code inside your test uses fake
-timers.
```

Comment:
I still stand by removing the `This should be used sporadically and not on a regular
basis since using it contains some overhead` sentence. Fake timers are nothing to be scared of when used correctly. And in some cases (let's say you're developing a calendar module) they need to be used in nearly all of your tests :)